Dockerfile

```diff
@@ -1,15 +1,12 @@
-FROM matomo:latest
-
-# Install system dependencies
-RUN apt-get update && apt-get upgrade -y 
-
-
-EXPOSE 80 443
-
-COPY config.ini.php /var/www/html/config/config.ini.php
-
-
+FROM matomo:3.13.6-apache
 
+# Add the EnvironmentVariables plugin
+COPY ./files/plugin-EnvironmentVariables-3.0.0/ /var/www/html/plugins/EnvironmentVariables
 
+# Preconfigure settings
+COPY ./files/config.ini.php /var/www/html/config/config.ini.php
 
+# Create mount point for EFS partition
+RUN mkdir -p /mnt/efs
```

Comment:
I'm curious how this worked before, but not enough to dig in on it ;)

Reply:
ha ha! the legacy AWS configuration mounts EFS to `/var/www/html/config` which